### PR TITLE
Feat - Namespace update

### PR DESCRIPTION
**Pull request by alecritson**
🤯

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <div align="center" height="400">
-    <img src="https://raw.githubusercontent.com/getcandy/getcandy/main/.github/assets/logo-light.svg#gh-dark-mode-only" alt="GetCandy" width="400">
-    <img src="https://raw.githubusercontent.com/getcandy/getcandy/main/.github/assets/logo.svg#gh-light-mode-only" alt="GetCandy" width="400">
+    <img src="https://raw.githubusercontent.com/lunar/lunar/main/.github/assets/logo-light.svg#gh-dark-mode-only" alt="GetCandy" width="400">
+    <img src="https://raw.githubusercontent.com/lunar/lunar/main/.github/assets/logo.svg#gh-light-mode-only" alt="GetCandy" width="400">
```
Comment (markmead) [inline README.md:3]: - Should this be `lunarphp/lunar`?
- Alt text says "GetCandy"

File: README.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 </div>
 
-[GetCandy](https://getcandy.io) is a set of Laravel packages that bring functionality akin to Shopify and other e-commerce platforms to Laravel. You have complete freedom to create your own storefront(s), but we've already done the hard work for you in the backend.
+[GetCandy](https://lunarphp.io) is a set of Laravel packages that bring functionality akin to Shopify and other e-commerce platforms to Laravel. You have complete freedom to create your own storefront(s), but we've already done the hard work for you in the backend.
 
 This repository serves as a monorepo for the main packages that make up GetCandy.
```
Comment (markmead) [inline README.md:8]: GetCandy.

File: README.md [hunk 3]
```diff
@@ -15,12 +15,12 @@ This repository serves as a monorepo for the main packages that make up GetCandy
 
 ## Documentation
 
-- [Full documentation](https://docs.getcandy.io/) - Includes in-depth guides on everything GetCandy
+- [Full documentation](https://docs.lunarphp.io/) - Includes in-depth guides on everything GetCandy
```
Comment (markmead) [inline README.md:18]: GetCandy.

File: README.md [hunk 4]
```diff
@@ -15,12 +15,12 @@ This repository serves as a monorepo for the main packages that make up GetCandy
...
 
 ## Community
 
 - [Join our discord server](https://discord.gg/v6qVWaf) and chat to the developers and people using GetCandy.
```
Comment (markmead) [inline README.md:22]: GetCandy.

File: docs/src/README.md [hunk 5]
```diff
@@ -1,22 +1,22 @@
 # Welcome to GetCandy!
```
Comment (markmead) [inline docs/src/README.md:1]: GetCandy.

File: docs/src/README.md [hunk 6]
```diff
@@ -1,22 +1,22 @@
...
 
-We're delighted you are considering GetCandy for your project. We've spent a lot of time developing this package to bring headless e-commerce functionality to Laravel.
+We're delighted you are considering Lunar for your project. We've spent a lot of time developing this package to bring headless e-commerce functionality to Laravel.
 
 ## What is GetCandy?
```
Comment (markmead) [inline docs/src/README.md:5]: GetCandy.

File: docs/src/contributing.md [hunk 7]
```diff
@@ -63,12 +63,12 @@ However, if you're not used to working with monorepo's and setting them up insid
 ::: tip Development Guide
 Here's a guide on how to set-up your development environent ready for contributing to GetCandy.
```
Comment (markmead) [inline docs/src/contributing.md:64]: GetCandy.

File: docs/src/extending/taxation.md [hunk 8]
```diff
@@ -4,7 +4,7 @@
 
 ## Overview
 
-Taxation is a tricky business and sometimes what GetCandy offers simply won't be enough, and we completely understand. This why Taxation is driver based, so you can add your own logic if you need to.
+Taxation is a tricky business and sometimes what Lunar offers simply won't be enough, and we completely understand. This why Taxation is driver based, so you can add your own logic if you need to.
 
 By default we have a `SystemTaxManager` which will use GetCandy's internal models and database as outlined above. If you need to write our own implementation, or if you're creating an add on for Tax, you can change the driver in the `config/taxes.php` config file.
```
Comment (markmead) [inline docs/src/extending/taxation.md:9]: GetCandy.

File: packages/core/LICENSE.md [hunk 9]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) GetCandy Ltd.
+Copyright (c) Lunar Ltd.
```
Comment (glennjacobs) [inline packages/core/LICENSE.md:3]: Needs changing to `LunarPHP Ltd`

File: packages/admin/LICENSE.md [hunk 10]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) GetCandy Ltd.
+Copyright (c) Lunar Ltd.
```
Comment (glennjacobs) [inline packages/admin/LICENSE.md]: Change to `LunarPHP Ltd`

**Review by glennjacobs — APPROVED** [review]

**Pull request merged**